src/Hedgehog/Seed.fs

```diff
@@ -46,24 +46,6 @@ module Seed =
     let [<Literal>] private goldenGamma : uint64 =
         0x9e3779b97f4a7c15UL
 
-    let private crashUnless (cond : bool) (msg : string) : unit =
-        if cond then
-            ()
-        else
-            failwith msg
-
-    /// Create a new 'Seed' from a 32-bit integer.
-    let ofInt32 (s0 : int) : Seed =
```

Comment:
Guess we didn't need this function anymore?

Reply:
There was a [bug in the SplitMix paper](https://github.com/hedgehogqa/haskell-hedgehog/issues/191) so after I've fixed it in Haskell I applied the changes in F#. I probably left this function in there by accident (my memory just can't go so many years back :confused:).

Reply:
Can you add to the comments in `Seed.fs` something about the bug in the original paper and link to [that haskell-hedgehog issue](https://github.com/hedgehogqa/haskell-hedgehog/issues/191)?